src/layouts/Navbar.tsx

```diff
@@ -45,10 +45,17 @@ const Navbar = ({ clubname, pages, links, exit }: Nav) => {
       {/* MOBILE */}
       <nav
         id="popup"
-        className="hidden flex w-screen h-screen p-2.5 z-50 justify-center items-center fixed bg-neutral-950">
-        <div className="flex flex-col min-w-[350px] w-full mx-2.5 justify-between items-center self-stretch rounded-xl border-2 border-white">
-          <div className="flex flex-col min-h-[350px] h-5/6 justify-center items-center gap-5">
-            <img src={HM_White_Transparent} className="h-20 w-20" />
+        className="hidden flex fixed w-screen h-screen p-2.5 z-50 bg-neutral-950">
+        <div className="flex flex-col min-w-[350px] w-full mx-2.5 rounded-xl border-2 border-white">
+          
+          <div className="flex flex-col w-[100%] items-end">
+            <button onClick={ShowPopup}>
+              <img className="w-6 mt-2.5 mr-2.5 text-white" src={close_button}></img>
```

Comment:
Add more spacing between close button and put logo to top left

![image](https://github.com/HackMelbourne/HackMelbourne.github.io/assets/30622247/00a5066b-7538-4b20-99a1-890ed31e4a6f)